core/src/main/java/io/servicecomb/core/provider/consumer/InvokerUtils.java

```diff
@@ -72,18 +76,23 @@ public static Response innerSyncInvoke(Invocation invocation) {
           String.format("invoke failed, %s", invocation.getOperationMeta().getMicroserviceQualifiedName());
       LOGGER.debug(msg, e);
       return Response.createConsumerFail(e);
+    } finally {
+      invocation.triggerFinishedEvent();
```

Comment:
seems we missed success /fail information

Reply:
create a new JIRA process this : https://issues.apache.org/jira/browse/SCB-150